cli/README.md

```diff
@@ -74,6 +74,10 @@ We love our bleeding edge tech. The amount of speed and, honestly, _fun_ that co
 
 The stated goal of `create-t3-app` is to provide the quickest way to start a new full-stack, typesafe web application. We take typesafety seriously in these parts as it improves our productivity and helps us ship fewer bugs. Any decision that compromises the typesafe nature of `create-t3-app` is a decision that should be made in a different project.
 
+### 4. Environment Variables
```

Comment:
I'm not sure where a note about env variables would go, but definitely not in the T3 Axioms

Reply:
I guess the most reasonable place to put something about .env vs .env.local would be here? https://create.t3.gg/en/usage/env-variables#using-environment-variables